orc8r/cloud/go/services/configurator/storage/sql.go

```diff
@@ -472,6 +472,12 @@ func (store *sqlConfiguratorStorage) LoadEntities(networkID string, filter Entit
 	}
 	sort.Slice(ret.Entities, func(i, j int) bool { return entComparator(ret.Entities[i], ret.Entities[j]) })
```

Comment:
Side note: update the doc.go of the configurator and/or configurator/storage pkgs to briefly mention the pagination functionality